projects/core/abstract/abstract-hint-options.ts

```diff
@@ -1,7 +1,7 @@
-import {TuiDirection, TuiHintModeT} from '@taiga-ui/core/types';
+import {TuiBrightness, TuiDirection, TuiHintModeT} from '@taiga-ui/core/types';
 
 export interface TuiAbstractHintOptions {
-    readonly mode: TuiHintModeT | null;
+    readonly mode: TuiHintModeT | TuiBrightness | null;
```

Comment:
We do not support whole `TuiBrightness`, only `onDark` which is already in `TuiHintModeT`.

Reply:
But when I change it

<img width="488" alt="image" src="https://user-images.githubusercontent.com/12021443/156872884-6e4cdcc6-9202-4177-b20c-5e0069c81434.png">

I have problems

<img width="1589" alt="image" src="https://user-images.githubusercontent.com/12021443/156872958-b7544d0c-65e3-430e-9ac0-a4b597221dd2.png">

Reply:
fixed